src/components/bar-chart/bar-chart.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BarChartComponent } from './bar-chart.component';
+import { MatCardModule } from '@angular/material/card';
+import { BarChartD3 } from '../../d3/bar-chart.d3';
+
+describe('BarChartComponent', () => {
+  let fixture: ComponentFixture<BarChartComponent>;
+  let component: BarChartComponent;
+  let barChartD3: BarChartD3;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        MatCardModule,
+      ],
+      declarations: [
+        BarChartComponent,
+      ],
+    });
+    fixture = TestBed.createComponent(BarChartComponent);
+    component = fixture.componentInstance;
+    // tslint:disable-next-line:no-string-literal
+    barChartD3 = component['barChartD3'];

```

Comment:
just curious, why can't you use dot access here?

Reply:
https://github.com/googleinterns/guide-doge/blob/master/src/components/line-chart/line-chart.component.spec.ts#L24

I guess it is a historical issue. I remember long time ago `lineChartD3` is private in LineChartComponent, but somehow it is changed to public now. I can fix both tests for line and bar (and pie) chart.

Reply:
FIxed in #110